Changed Product Name

jamesche75 opened the pull request:
Changed "Web Apps for Containers" to "Web App for Containers". There are also links in here to other docs that use "Apps" instead of "App". I didn't change those because it would break the links, but I will do a PR on those docs.

azurecla commented:
Hi __@huangcd__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

davidwrede commented:
Thanks huangcd for catching this typo! We appreciate your contribution in making our documentation better.

- Dave

jamesche75 committed:
Changed Product Name

Changed "Web Apps for Containers" to "Web App for Containers". There are also links in here to other docs that use "Apps" instead of "App". I didn't change those because it would break the links, but I will do a PR on those docs.
